Sources/UIDeviceExtensions.swift

```diff
@@ -85,6 +85,18 @@ extension UIDevice {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
 
+    /// EZSE: Returns path of documents directory
+    class func documentsDirectoryPath() -> String {
```

Comment:
No I dont think its needed. I added CGPoint extensions before. So this project does not stop anyone.